web/client/plugins/Identify.jsx

```diff
@@ -94,28 +87,18 @@ const identifyDefaultProps = defaultProps({
     reverseGeocodeData: {},
     enableRevGeocode: true,
     wrapRevGeocode: false,
-    queryableLayersFilter: MapInfoUtils.defaultQueryableFilter,
     style: {},
     point: {},
     layer: null,
     map: {},
     layers: [],
-    maxItems: 10,
-    excludeParams: ["SLD_BODY"],
-    includeOptions: [
-        "buffer",
-        "cql_filter",
-        "filter",
-        "propertyName"
-    ],
     panelClassName: "modal-dialog info-panel modal-content",
     headerClassName: "modal-header",
     bodyClassName: "modal-body info-wrap",
     dock: true,
     headerGlyph: "",
     closeGlyph: "1-close",
     className: "square-button",
-    allowMultiselection: false,
```

Comment:
I would not remove this without an explicit  reason.

Reply:
I partially maintained it (that it means that is not configurable anymore, but the code base for this flag is still maintained and tested). Ideally, after this changes, it should go in the application state. 
If you want I can finalize the support for this flag in the mapInfo state.

Reply:
See this https://github.com/geosolutions-it/MapStore2/pull/3616/files#diff-e7312b552c083e1cbff765fe521d4d0bR82